---
title: "Neural Connectome of the Ctenophore Statocyst"
subtitle: |
  **Kei Jokura**$^{1,2,3,4,5*}$, **Sanja Jasek**$^{1,2,6}$, **Lara Kewalow**$^{6}$, **Pawel Burkhardt**$^{7}$, **Gáspár Jékely**$^{1,2,6,*}$  
  <br><br>  
  $^{1}$ Living Systems Institute, University of Exeter, Exeter, EX4 4QD, United Kingdom  
  <br>
  $^{2}$ Biosciences, Faculty of Health and Life Sciences, University of Exeter, Exeter EX4 4QD, UK
  <br>  
  $^{3}$ Grass Laboratory, Marine Biological Laboratory, Woods Hole, MA 02543, USA  
  <br> 
  $^{4}$ Exploratory Research Center on Life and Living Systems (ExCELLS), Okazaki, 444-8787, Japan  
  <br>  
  $^{5}$ National Institute for Basic Biology (NIBB), Okazaki, 444-8585, Japan  
  <br> 
  $^{6}$ Heidelberg University, Centre for Organismal Studies (COS), 69120 Heidelberg, Germany  
  <br>
  $^{7}$ Michael Sars Centre, University of Bergen, Norway
  <br><br>  
  **$^{*}$ Correspondence:**  
  [jokura@nibb.ac.jp](mailto:jokura@nibb.ac.jp),  
  [gaspar.jekely@cos.uni-heidelberg.de](mailto:gaspar.jekely@cos.uni-heidelberg.de)
mainfont: Arial
monofont: DejaVu Sans Mono #Times New Roman
fontsize: 11pt
linestretch: 1.2
linkcolor: blue
documentclass: article
max-width: 100%
format:
  html:
    embed-resources: true 
  pdf:
    pdf-engine: xelatex
    number-sections: false
    margin-left: 0.8in
    margin-right: 0.8in
    margin-top: 0.8in
    margin-bottom: 0.8in
    toc: false
    keep-tex: true
    include-in-header:
      text: |
        \usepackage{lineno}
    include-before-body:
      text: |
        \linenumbers
bibliography: references.bib #add bibliography in this file
csl: elife.csl #change according to journal
---

## Abstract

Ctenophores possess a unique gravity receptor (statocyst) in their aboral organ formed by four clusters of ciliated balancer cells that collectively support a statolyth. During reorientation, differential load on the balancer cilia leads to altered beating of the ciliated comb rows to elicit turns. To study the neural bases of gravity sensing, we imaged by volume electron microscopy (vEM) the aboral organ of the ctenophore *Mnemiopsis leidyi*. We reconstructed 909 cells, including syncytial neurons that form a nerve net. The syncytial neurons synapse on the balancer cells and also form reciprocal connection with the bridge cells that span the statocyst. High-speed imaging revealed that balancer cilia beat and arrest in a coordinated manner but with differences between the sagittal and tentacular planes of the animal, reflecting nerve-net organisation. Our results suggest a coordinating rather than effector function for the nerve net and inform our understanding of the diversity of nervous-systems organisation across animals.

## Introduction

Ctenophores, or comb jellies, are gelatinous marine animals that actively swim by beating rows of fused cilia known as comb plates. These coordinated ciliary movements not only generate propulsion but also allow precise control of body orientation and directional changes within the water column. Despite lacking a centralized nervous system, ctenophores exhibit sophisticated and dynamic behavioral patterns and respond to a broad range of external stimuli, suggesting the presence of an evolutionarily unique and functionally integrated sensorimotor system.
Several components of the ctenophore nervous system have been described, including a subepithelial nerve net (SNN), mesogleal neurons, tentacular nerves, and elements of nervous system in the aboral organ [@Hertwig; @Hernandez-Nicaise; @Jager]. These neural structures are characterized by unique features such as a syncytial architecture [@Burkhardt] and an extensive repertoire of lineage-specific neuropeptides not found in other metazoans [@Sachkova2021; @Hayakawa2022]. Notably, early single-cell transcriptomic studies failed to identify canonical neuronal clusters using conventional markers [@Sebe-Pedros2018]. It was only through the use of pro-neuropeptide-based markers that distinct neural cell types were subsequently identified [@Sachkova2021; @Hayakawa2022].
In parallel, phylogenomic and chromosomal synteny analyses have suggested that ctenophores may represent the sister group to all other animals [@Ryan2013; @Whelan2015; @Li2021; @Schultz2023], raising the possibility that their nervous system evolved independently from those of other metazoans [@Moroz]. This hypothesis positions ctenophores as a key lineage for re-evaluating the origins and diversity of neural systems.
However, without a detailed understanding of how ctenophore neurons are spatially organized and interconnected, it remains unclear how their nervous system gives rise to behavior. Clarifying this relationship between neural architecture and behavioral control is essential to uncovering the logic of ctenophore neural function.

To address the lack of structural understanding of how ctenophores control behavior, we focused on a unique sensory structure—the aboral organ. This gravity-sensing organ contains a statocyst supported by four mechanosensory balancer cells, which act as pacemakers for comb plate beating and posture control [@Horiddge1965; @Tamm1980]. Ultrastructural and immunohistochemical studies have revealed that the aboral organ contains diverse sensory and neural components, including a distinct neural structure termed the deep nerve net [@aronova1974electron; @Hernandez-Nicaise1968; @Jager2011].
A key feature of balancer cilia is their bidirectional, mechanoresponsive control of beat frequency. Depending on the direction of deflection and the animal’s geotactic state, balancer cilia may either increase or decrease their activity, indicating a dynamic response system not fixed like vertebrate hair cells [@Horridge1965; @Tamm1982; @Tamm2014]. 
Previous studies have implicated membrane potential, intracellular calcium concentration, and synaptic input in modulating this response [@Lowe1997]. In particular, deflection-induced excitation can be abolished by calcium channel inhibitors or calcium-free conditions [@Lowe1997], while high extracellular K⁺ depolarizes and activates cilia in the absence of mechanical stimuli [@Lowe1997]. Local calcium application further suggests that Ca²⁺ influx at the ciliary base may be required for the excitatory response [@Lowe1997]. These findings support the hypothesis that neural modulation plays a critical role in shaping balancer cell output [@Lowe1997].

To investigate how balancer cilia are modulated within the aboral organ, we combined volume electron microscopy and high-speed imaging in the ctenophore *Mnemiopsis leidyi*. Through connectomic reconstruction, we characterized the synaptic architecture of the aboral nerve net, and functional imaging revealed correlated activity patterns in balancer cilia. These results suggest that the aboral nerve net plays a central role in integrating sensory input and coordinating motor output.

## Results

### Volume EM reconstruction of the *Mnemiopsis* aboral organ

For vEM analysis, we used a five-day-old cydippid larva of *Mnemiopsis leidyi*. To obtain high-quality ultrastructural preservation, we used high pressure freezing followed by freeze substitution and resin embedding (Epon). The specimen was sectioned from the aboral tip and we collected approximately 1,000 ultra-thin (50 nm) serial sections as ribbons. The sections were imaged in a Zeiss Gemini 500 scanning electron microscope at 2.0 nm/pixel (xy). We imaged only the region containing the aboral organ. Subsequently, 620 of these images were stitched and aligned in TrakEM2 [@cardona2012]. The final vEM dataset encompassed a volume of 60 μm × 40 μm × 30 μm. We skeletonised and annotated all cells in this dataset in CATMAID [@Saalfeld2009], resulting in a reconstruction of 909 cells with nuclei and cell bodies intact. Most of the cells, including the balancers and the aboral nerve net neurons are fully contained within the volume, due to the compact and mostly self-contained organisation of the aboral organ.

In cells containing cilia, we also traced cilia along their length and annotated all basal bodies. For neuronal skeletonization, nodes were placed to interconnect the profiles of the same neuron’s processes across layers, extending the skeleton until all branches were fully traced. All nodes relevant to synaptic sites were tagged, and skeletons were named and assigned multi-level annotations. As described later, some neurons formed loop-like structures (anastomosed neurons), wherein separated branches often rejoined either the main branch or smaller branches [@Burkhardt2023]. In such cases, branch nodes were placed near the closest existing node and annotated accordingly (CATMAID only supports skeleton trees). The entire skeletonized volume was composed of 134,591 nodes. 88 fragments could not be attached to somata-associated skeletons. Most of these fragments represent short skeletal branches that could not be traced beyond gaps or low-quality layers.

Next, we divided the entire aboral organ into four broad quadrants (Q1-Q4) to facilitate grouping the identified cells (Figure 1F) [@Martindale1999]. The general body plan of ctenophores, when viewed from the aboral side, exhibits biradial symmetry around the anal pores [@Martindale1999]. This symmetry corresponds to the four blastomeres present at the four-cell stage during early embryonic development [@Martindale1999]. We assigned each of the 909 skeletonised cells to one of the four quadrants.

![**Figure 1. Morphology of the aboral organ in *Mnemiopsis leidyi***\
(A) Whole-body image of a 5-day-old *M. leidyi* cydippid larva in lateral view. The boxed region indicates the aboral organ (ao). Scale bar: 100 µm.\
(B) Schematic diagram of the aboral view of a cydippid larva. The two-tone coloration represents the biradial symmetry of the body. For convenience, the body is divided into four quadrants, designated as the first (Q1) through fourth (Q4) quadrants. The two primary viewing angles are referred to as the sagittal plane (S) and the tentacular plane (T). Abbreviations: ao, aboral organ; cg, ciliated groove; cr, comb plates; tb, tentacle bulbs.\
(C) Aboral views of the aboral organ highlighting its spatial organization. The left panel presents a schematic representation of the aboral organ, illustrating the four body quadrants (Q1–Q4), color-coded consistently with panel (B). The right panel shows a corresponding differential interference contrast (DIC) image from the same perspective. The region enclosed by the dotted line delineates the boundary of the aboral organ. The sagittal (S) and tentacular (T) arrows indicate the viewing directions corresponding to those planes. Scale bar: 50 µm. bal, balancer; cg, ciliated grooves; li, lithocyte.\
(D) Lateral views of the aboral organ in two orthogonal planes captured using DIC microscopy. The left panel shows a lateral view in the sagittal (S) plane, while the right panel displays a view in the tentacular (T) plane. Balancer cilia (bal) and lithocytes (li) are enclosed within the dome (do). Scale bar: 50 µm. A–O indicates the aboral–oral body axis.\
(E) Schematic representations of the aboral organ in sagittal and tentacular planes.
The left panel shows a lateral schematic in the sagittal plane (S), and the right panel shows the same in the tentacular plane (T), corresponding to the views in panel (D). Colors match the quadrant scheme shown in panels (B) and (C).\
(F) Example of cell tracing using the collaborative annotation toolkit CATMAID for large-scale electron microscopy image datasets. The spherical objects indicate nuclear positions, while the lines represent the traced cell centers. Scale bar: 10 µm.\
(G) Three-dimensional reconstruction of cells composing the aboral organ, displayed from different perspectives. The left panel presents an aboral view of the aboral organ. The middle panel shows a sagittal plane view, while the right panel provides a tentacular plane view. Cells are color-coded according to their respective quadrants. Lithocytes (li) are represented as three gray spheres, while balancers (bal) are depicted as gray lines. Scale bar: 25 µm.](figures/Figure1.png)

### An aboral synaptic nerve net of syncytial neurons

Classical neural staining techniques did not provide clear images of neurons at the aboral organ. However, ultrastructural studies revealed morphological evidence for neural elements, including synapses that are located on the epithelial floor of the aboral organ—a region where epithelial cells are densely packed and form a floor-like structure [@Horridge_1964; @Hernandez_Nicaise_1973]. In our dataset, we identified synapses based on the previously described pre-synaptic triad morphology consisting of a single layer of vesicles, a cisterna of smooth endoplasmic reticulum and an associated mitochondrion. At synaptic sites, we marked mitochondria as synaptic nodes (orange in CATMAID) and connected this node to the nearest node in postsynaptic cells across the regions where synaptic vesicles aligned with the presynaptic cell's membrane (light blue arrows in CATMAID) (Supplementary Figure S2A). We could not detect specialized post-synaptic structures, in agreement with previous studies [@Hernandez_Nicaise_1973]. Synapses were identified as either monoadic or polyadic, with one pre-synaptic neuron forming connections with one or multiple post-synaptic cells (Supplementary Figure S2B).

We reconstructed three aboral nerve-net (ANN) neurons, each with a syncytial morphology (Figure 2A and B). Each ANN had multiple nuclei (ranging from two to five per neuron), with anastomosing membranes (Supplementary Figure S2C). These neurons are distinct from the syncytial subepithelial nerve net (SNN) neurons with blebbed morphology previously identified by vEM in the body wall of *Mnemiopsis* [@Burkhardt2023]. Our results reveal a second type of syncytial neurons in ctenophores (Supplementary Figure S2D). 

Furthermore, ANNs are also different from other neurons previously reported by EM, including mesogleal neurons and ciliated sensory cells that synapse on the SNN (types 1-4)[@Burkhardt2023]. Based on their distinct size and position, we classified the ANNs into two types. The first type is a single large neuron (ANN_Q1-4) spans all four quadrants (Q1-Q4) and has four (possibly five) nuclei. This neuron has 150 presynaptic sites. The second type consists of two neurons (ANN_Q1Q2 and ANN_Q3Q4), each spanning two quadrants and containing two nuclei. ANN_Q1Q2 had 119 presynaptic structures, whereas ANN_Q3Q4 had 68 presynaptic structures.

![**Figure 2. Organisation of the aboral synaptic nerve net**\
(A) 3D reconstruction of the aboral nerve net (ANN) Q1Q2Q3Q4 (Q1-4), which spans all four quadrants and contains multiple nuclei (blue). The spheres represent the positions of individual nuclei. The left panel shows a aboral view of the aboral organ, the middle panel presents a sagittal plane view, and the right panel provides a tentacular plane view.\
(B) 3D reconstruction of two ANN Q1Q2 (pink) and Q3Q4 (orange), each spanning two quadrants and containing multiple nuclei. The spheres indicate the positions of the nuclei. The left panel shows a aboral view of the aboral organ, the middle panel presents a sagittal plane view, and the right panel provides a tentacular plane view.\
(C) Graph showing the number of mitochondria per cell and the proportion of mitochondria involved in synapse formation for each cell type. Green represents mitochondria associated with synapses, while gray represents mitochondria not involved in synapse formation. ANN has a significantly higher number of mitochondria compared to other cell types, with most of them forming synapses. Except for lithocytes, each cell type has a sample size of at least three cells. The abbreviations are as follows: bal (balancer), brg (bridge), bsl (bristle), cg (ciliated groove), dv (dense vesicle cells), imc (intra-multiciliated cells), la (lamellate bodies), li (lithocytes), pl (plumose), and ef (epithelial floor cells).\
(D) Localization of mitochondria within ANN Q1-4. Red indicates mitochondria associated with the presynaptic triad structures, yellow marks mitochondria containing synaptic vesicles but lacking a clearly defined presynaptic triad, blue represents mitochondria with unclear synaptic vesicles, and black denotes mitochondria where no synaptic vesicles were identified.\
(E) Representative electron micrographs of presynaptic triad structures observed in the dataset and their schematic diagrams. The left diagram illustrates synaptic projections from the center ANN (ANN Q1-4) to the lateral ANN (ANN Q3Q4), while the right diagram shows synaptic projections from the lateral ANN (ANN Q1Q2) to the center ANN (ANN Q1-4). The abbreviations are as follows: mi (mitochondrion), er (endoplasmic reticulum), and sv (synaptic vesicles).\
(F) Plot showing the three-dimensional positions of synapses. Synapses from the center ANN to the lateral ANN are shown in magenta, while synapses from the lateral ANN to the center ANN are shown in cyan. Blue dots indicate the locations of autapses within ANN Q1-4.](figures/Figure2.png)

### Synaptic connectome of the gravisensory organ

Our synapse annotation revealed synapses between the ANN neurons and synapses that ANN_Q1-4 forms on itself (autapses). The ANN neurons also form synapses on the gravity-sensing balancer cells.

Balancer cells are monociliated cells with long motile cilia that form four bundles of compound cilia, one in each quadrant. These bundles come together at the centre of the aboral organ to support the cellular mass of the statolyth. The bending of the cilia and the position of somata differed clearly when viewed laterally from the sagittal or the tentacular plane (Figure 3B). In each quadrant, there were between 28-37 balancer cells (Q1: 37; Q2: 32; Q3: 32; Q4: 28). Each cell contained 3 to 10 <!--# check --> mitochondria.

ANN_Q1-4 formed synapses on balancer cells in all four quadrants (on 7 cells in Q1, 11 cells in Q2, 6 cells in Q3 and 10 cells in Q4) while ANN_Q1Q2 and ANN_Q3Q4 synapsed to balancer cells in their respective quadrants (ANN_Q1Q2 on 6/37 cells in Q1 and 8/32 cells in Q2; ANN_Q3Q4 on 1/32 cells in Q3 and 5/28 cells in Q4). Some balancer cells received inputs from both ANN_Q1-4 and either of ANN_Q1Q2 or ANN_Q3Q4. While previous studies suggested the presence of afferent synapses from balancer cells to neurons [@Hernandez-Nicaise1974], our data did not reveal any presynaptic sites in balancer cells.

The second group of cells that form synaptic contact with the ANN were the bridge cells. Bridge cells, first described by Tamm et al. in 2002 [@Tamm_2002], are characterized by bundles of elongated processes filled with microtubules that arch over the epithelial layer, resembling a bridge. Their somata are located at the base of the paired balancer-cell clusters along the tentacle surface and extend across the sagittal plane towards the base of the opposite balancer cells. Bridge cells form two distinct cell groups across the sagittal plane, in the Q1Q2 and Q3Q4 regions. In the Q1Q2 region, we identified 14 bridge cells, in Q3Q4, 12 cells.

Bridge cells had presynaptic sites with the typical presynaptic triad structure near 30% of their mitochondria. These bridge-cell synapses were formed on ANN neurons or other bridge cells. Bridge cells in the Q1Q2 region formed synapses on ANN_Q1Q2 (3 cells) or on both ANN_Q1Q2 and ANN_Q1-4 (2 cells). Bridge cells in Q3Q4 synapsed on ANN_Q3Q4 (1 cell) or ANN_Q1-4 (6 cells).

Nearly all bridge cells (25/26) also received synaptic input from ANNs. Bridge cells in Q1Q2 received inputs from ANN_Q1Q2 (11 cells), ANN_Q1-4 (1 cell), or both (2 cells). In Q3Q4, synapses on bridge cells were from ANN_Q3Q4 (1 cell), ANN_Q1-4 (7 cells), or both (1 cell).

In both The Q1Q2 and the Q3Q4 regions, bridge cells also formed synapses with adjacent bridge cells. However, we found no synapses across the sagittal plane to bridge cells in the opposite region.

To analyze the synaptic connectivity graph of the balancer organ and the ANN, we grouped cells of the same type and in the same region into a single node, summed the number of synapses, and laid out the network to reflect the anatomy of the four quadrants (Figure 3H). The network is characterised by feedback connections among ANN neurons and between ANN and bridge cells with a clear rotational (or mirror) symmetry. Q1Q2 and Q3Q4 also form separate local circuits centered around their regional ANN neurons while ANN_Q1-4 mediates global connectivity across the entire balancer organ. Notably, there were no synapses from the mechanosensory balancer cells to the ANN or from ANN to motor cells (e.g. ciliary groove), contrary to our initial expectation of a sensory-motor or input-output model of balancer function.

The ANN neurons also formed synapses on other cell types in the aboral organ, including the dense-vesicle cells, EF cells and several non-ciliated, monociliated or biciliated cells. These cell types and synaptic connections are outside the gravisensory organ and will be described elsewhere.

![**Figure 3. Reconstruction of the balancers and bridges that receives synapses from the ANN.**\
(A) Graph showing the number of synaptic outputs from the ANNs (ANN Q1-4: blue, ANN Q1Q2: orange, ANN Q3Q4: pink) circuit for each cell type. The balancer and bridge cell types receive relatively high outputs from the ANNs circuit. ANN represents the number of synapses within the ANNs circuit. The abbreviations are as follows: bal (balancer), brg (bridge), bsl (bristle), cg (ciliated groove), dv (dense vesicle cells), imc (intra-multiciliated cells), la (lamellate bodies), li (lithocytes), pl (plumose), and ef (epithelial floor cells).\
(B) Connectivity map of three ANN neurons (Q1-4 in blue, Q1Q2 in pink, and Q3Q4 in orange) and balancer ciliated cells (light blue). The thickness of the arrows and the numbers correspond to the number of synapses. Synaptic structures from the same neuron targeting the same balancer cell are grouped into hexagons, with the number of cells added to the label.\
(C) 3D reconstruction of bridge cells spanning the Q1Q2 and Q3Q4 quadrants. The Q1Q2-side bridge cells (8 cells) are shown in blue, while the Q3Q4-side bridge cells (6 cells) are shown in xxx color. The morphology of individual bridge cells extending across opposite quadrant regions is depicted. The spheres represent the positions of individual nuclei. The left panel shows an aboral view of the aboral organ, the middle panel presents a sagittal plane view, and the right panel provides a tentacular plane view.\
(D) Mitochondrial localization within bridge cells and associated presynaptic triad structures. Red indicates mitochondria associated with presynaptic triad structures, yellow marks mitochondria containing synaptic vesicles but lacking a clearly defined presynaptic triad, blue represents mitochondria with unclear synaptic vesicles, and black denotes mitochondria where no synaptic vesicles were identified. The left panel shows a dorsal view of the aboral organ, the middle panel presents a sagittal plane view, and the right panel provides a tentacular plane view.\
(E) Synaptic connections from ANN neurons to balancer ciliated cells. The positions of synapses from ANNs to balancers (magenta) are indicated. The left panel presents an aboral view, while the right panel shows a lateral view and a tentacular plane view. Balancer ciliated cells are depicted in light gray.\
(F) Synaptic connections between ANNs and bridge cells. The positions of synapses from ANNs to bridge cells (magenta) and from bridge cells to ANNs (light blue) are indicated. The left panel shows an aboral view, while the right panel presents a tentacular plane lateral view. Bridge cells are shown in light gray.\
(G) Connectivity matrix of the gravity-sensing neural circuit. Columns represent presynaptic cell groups, while rows represent postsynaptic cell groups. The numbers and varying shades of blue correspond to the number of synapses.\
(H) Complete connectivity map of the gravity-sensing neural circuit. Cells belonging to the same group are enclosed in hexagons, and the number of cells is added to their labels. The thickness of the arrows and the numerical values indicate the number of synapses. ANN Q1-4 are shown in blue, ANN Q1Q2 in orange, ANNs in Q3Q4 in pink, balancer cell groups in gray, and bridge cell groups in yellow.](figures/Figure3.png)

### Dynamics of balancer cilia imply a coordinating function for the nerve net

To investigate the regulation of balancer function, we carried out a high-speed video-microscope analysis of balancer cilia. Previous studies [@Tamm1980; @Tamm1982; @Lowe1997] have established that balancer cilia function as mechanoreceptors, with their beating frequency modulated by inclination. Tamm also suggested that differences in statolith morphology and the shape of balancer cilia between the tentacular and sagittal planes could lead to different forces exerted on cilia by the statolith [@Tamm_2014; @Tamm2015]. To further explore this, we used a tilted microscope with a vertical stage where we mounted immobilised cydippid larvae with their aboral-oral axis aligned in different orientations relative to the gravity vector. We then compared larvae oriented in different angles and either with their sagittal or tentacular plane parallel to the sample stage.

In larvae with their sagittal plane facing the objective, we could compared balancer-cilia movements between Q1(4) and Q2(3). In other larvae oriented in the tentacular plane, we could simultaneously image Q1(3) and Q2(4). We used a high-speed camera and recorded at 100 fps for 2 minutes (12,000 frames). To analyse ciliary beating, we selected regions of interest (ROIs) in areas where brightness changes indicated ciliary beating. Ciliary beating was both manually quantified and plotted as kymographs.

During the two-minute recordings, balancer cilia could beat fast, slow, or exhibit abrupt stops (arrest) and start moving again (re-beat). Occasionally, large body-contractions moved the entire cydippid out of frame, and data from these episods were excluded. We focused on three metrics for inter-quadrant comparison: (1) the timing of ciliary arrest, (2) the timing of re-beat after an arrested phase, and (3) ciliary beat frequency.

We found that arrest and re-beat events were synchronized between balancers across the sagittal plane. However, while the timing of re-beat was also near-simultaneous along the tentacular plane, arrest timing was offset by up to 2.16 seconds (Figure 4D). Overall, these data reveal that arrests are only coordinated between Q1-Q2 and Q3-Q4, whereas re-beat is coordinated over all four quadrants.

As shown in Figure 4E, beat frequencies of left and right balancers fluctuated substantially in both orientations. However, while left-right activity was largely synchronized in sagittal pairs (Figure 4E, left), tentacular pairs (Figure 4E, right) showed strikingly periodic alternations between left and right balancer activity. This pattern suggested an offset in timing during arrest or re-beat events between tentacular balancers.

To quantify this, we calculated a rolling Pearson correlation between left and right balancer activity in each plane. The resulting mean correlations were significantly higher in the tentacular plane compared to the sagittal plane (Figure 4F; t-test, p = 0.014). This result indicates that tentacular balancers beat in a more coordinated, mirror-symmetric manner, whereas sagittal balancers exhibit greater variability between sides.

Evaluating these data in light of the circuit diagram suggests that shared ANN inputs (ANN_Q1Q2 or ANN_Q3Q4) to a pair of balancers along the sagittal plane may underlie their synchronized arrests. In contrast, in the tentacular plane, separate ANNs innervate the balancer pairs. At the same time, the ANN_Q1-4 neuron synapses on all four balancers, hinting at a neural substrate for their synchronised re-beat.

Notably, bridge neurons associated with each quadrant region (Q1Q2 or Q3Q4) extend their processes across the midline, forming connections that span between opposite quadrant domains when viewed along the tentacular plane. This morphology suggests a potential role for bridge neurons in modulating inter-quadrant coordination. By linking ANN activity across orthogonal planes, these neurons may contribute to the dynamic regulation of CBF patterns across the whole system.

![**Figure 4. Differences in Balancer Cilia Movement Control Due to Variations in the Neural Circuit Pathways**\
(A) Schematic diagram of differential interference contrast (DIC) microscopy setup for imaging balancer cilia movement. The microscope was tilted 90 degrees so that the stage was positioned vertically. A monochrome CMOS camera sensitive to near-infrared (NIR) light was used, synchronized with an 850 nm strobe light source. The movement of the balancer cilia was recorded using a 40× objective lens.\
(B) Representative DIC images of the aboral organ viewed along the sagittal (left) and tentacular (right) planes. Arrowheads indicate the balancer cilia selected for kymograph-based analysis in each orientation.\
(C) Representative kymographs showing arrest and re-beat events of balancer cilia. Left and right balancer cilia were simultaneously recorded in either the sagittal (left) or tentacular plane (right). Arrowheads mark the onset of arrest (open) and re-beat (filled); horizontal arrows represent 1-second intervals.\
(D) Boxplots showing time differences in the onset of arrest (white boxes) and re-beat (gray boxes) between left and right balancer cilia. Colors of the box outlines indicate the imaging plane: sagittal (blue) or tentacular (orange). Each dot represents a single larva.\
(E) Bar plots showing ciliary beat frequency (CBF) dynamics of left and right balancers (top and bottom, respectively) recorded from sagittal (left) and tentacular (right) planes. Positive and negative values represent opposite sides within each plane.\
(F) Mean rolling Pearson correlation of left–right balancer CBF calculated with a 20-frame window. Each dot represents an individual larva. Balancers on the tentacular plane showed significantly higher correlation than those on the sagittal plane (t-test, p = 0.014).](figures/Figure4.png)

## Discussion

In this study, we generated a comprehensive neural map of the aboral organ of a five-day-old *Mnemiopsis leidyi*. We described a novel type of syncytial neuron and analysed its synaptic connections to balancer-cell groups across the four quadrants of the aboral organ. Given that all neurons are intrinsic to the aboral organ and our EM volume covers the entire organ, we could recover a complete circuit map, the first for a ctenophore. Contrary to our expectations, the nerve net has no motor connections to effector organs, such as the ciliary grooves or comb rows. Our high-speed imaging suggests that the nerve net instead coordinates the activity of balancer cilia. This agrees with a hydrodynamic model of graviorientation <!-- add ref to Tamm -->, with balancer cilia directly responding to statolyth load and acting as pacemakers for the comb rows via the ciliated grooves. The nerve net could in turn tune or reset sensitivites and valence of balancer cells, in a coordinatory role, as we will discuss below.

![**Figure 5. Comparison of Neural Circuits Regulating Ciliary Movement**\
(A) Each neuron represented by a circle is color-coded to indicate putative homologous cell types involved in ciliary movement regulation. Ciliated cells are shown in gray. Synapses are indicated by arrows, with magenta representing synapses that induce ciliary arrest and blue representing synapses that induce ciliary re-beat or an increase in ciliary beating frequency.\
(Left) Gravity-sensing neural circuit of *M. leidyi*, as revealed in this study. Bridge cells (yellow squares) are suggested to be electrically coupled (indicated by yellow zigzag lines), implying their potential involvement in feedback mechanisms between neurons and ciliated cells.\
(Righr) Neural circuit regulating prototroch ciliary movement in *P. dumerilii*. Serotonergic neurons (Ser-h1 neurons, blue) activate ciliary movement, while cholinergic neurons (MC neurons, magenta) induce ciliary arrest. Arrows directed toward ciliated cells in the head prototroch are color-coded accordingly.](figures/Figure5.png)

### A New Type of Syncytial Neuron

We identified three syncytial neurons with multiple nuclei in the aboral organ of the cydippid larva. Burkhurdt et al. previously reported syncytial neurons in the sub-epithelial nerve net (SNN) of a one-day-old larva [@Burkhardt2023]. The syncytial neurons in the aboral organ did not exhibit the characteristic "pearl necklace-like structure" of the SNN neurons. Instead, their morphology was more reminiscent of fiber cells found in Placozoa [@Mayorova2021], with cell projections extending through intercellular spaces. In our volume, we also identified two SNN-like neurons with a pearl necklace-like morphology, indicating that the differences between ANN and SNN are not due to differences in stage or sample preparation. This confirms the ANN neurons as a second distinct type of syncytial neuron in ctenophores.

### Functional Diversification of the Aboral Syncytial Neurons

The three ANN neurons form two functional types based on their spatial arrangement and differential innervation of balancer cells. 

In the this study, we found that three neurons are closely positioned within the confined space of the aboral organ, potentially enabling a single neuron to control distant ciliary movements in a synchronized manner. Moreover, the presence of neurons projecting to the same ciliated cells in distinct patterns suggests that these neurons may serve opposing functions—one inhibiting ciliary activity while the other promotes it, or one modulating ciliary beat frequency. While our synaptic ultrastructure analysis did not allow us to distinguish excitatory from inhibitory neurons, future electrophysiological investigations will be essential to elucidate their functional properties.


In metazoans, a single neuron can directly provide synaptic input to multiple ciliated cells at a distance, synchronizing their movements. For instance, in larvae of the annelid *Platynereis dumerilii*, a cholinergic motor neuron (MC) innervates multiple prototroch ciliated cells, synchronizing their motion such that activation of the neuron leads to simultaneous arrest of ciliary beating [@Veraszto2017]. Additionally, a pair of serotonergic neurons, designated Ser-h1, extend axons that cross at the midline; the left Ser-h1_l predominantly innervates right-sided ciliated cells, while the right Ser-h1_r mainly projects to the left-side ciliated cells. Activation of Ser-h1 induces ciliary beating initiation and increases beat frequency.

### Paracrine Signaling and Synaptic Transmission from Sensory to Neural Cells

Previous electron microscopy studies have suggested the presence of various sensory cells, including photoreceptor and mechanoreceptor cells, within the aboral organ of ctenophores [@HORRIDGE1965; @vinnikov1974e; @Krisch1973; Aronova1974]. By comparing our dataset with previously reported ultrastructural data, we successfully identified several sensory cells within the aboral organ. However, notably, we did not find any clear presynaptic structures between these sensory cells and either neural or other cellular targets. In prior studies, immunostaining for specific neuropeptides in ctenophores has revealed positive signals in distinct cell populations within the aboral organ [@Sachkova2021; @Hayakawa2022]. These findings suggest that many sensory cells in the aboral organ may primarily rely on neuropeptide-mediated paracrine signaling rather than direct synaptic connections, supporting the chemical brain hypothesis [@Jékely2021]. While paracrine transmission is generally slower than synaptic transmission, it allows for a greater diversity of information processing.

Conversely, previous research has demonstrated that mechanoreceptive sensory cells equipped with cilia (referred to as type 3 and type 4 cells) in ctenophores form direct synaptic connections with the sub-epithelial nerve net (SNN) and mesogleal neurons [@Burkhardt2023]. Based on their ultrastructural features, these sensory cells are believed to respond to physical stimuli such as water flow, vibrations, and direct contact, transmitting information rapidly to cilia and muscle cells [@Hertwig1880; @Chun1880; @Eimer1880].

Surprisingly, our study identified bridge cells as the primary cell type forming synaptic inputs onto neurons in the aboral organ. Bridge cells were first described by Tamm et al. based on their morphological characteristics, with cellular projections extending to the basal region of balancer cilia [@Tamm_2002]. Our findings suggest that bridge cells are electrically coupled to balancer cilia and provide rapid feedback to neurons via synaptic transmission. Future studies analyzing differential responses to various sensory stimuli will not only elucidate the functional roles of bridge cells but also provide insights into the evolutionary relationship between transmission speed and synapse development.

### Circuit Variability as a Driver of Behavioral Diversity

Our study suggests that differences in neural circuits contribute to variations in the timing of ciliary arrest, which may play a role in modulating swimming direction in ctenophores [@Satterlie2015a]. This finding provides evidence that increased circuit complexity contributes to behavioral diversification. By integrating multiple neurons and altering circuit structure, information can be branched, allowing for diverse motor patterns within the framework of fast synaptic transmission.

The neural cells identified in the aboral organ exhibit a divergent feedforward connectivity pattern, where a single neuron forms presynaptic connections with multiple downstream targets. Such a circuit design minimizes unwanted signal interference, reduces processing delays, and enables relatively fast response execution [@Luo2021]. Moreover, the presence of multiple divergent feedforward circuits suggests that slight variations in the synchronization and phase delay of ciliary beating (within the range of a few hundred milliseconds) may have evolved as a mechanism for generating behavioral diversity.

From a broader neuroanatomical perspective, nervous system structures across various animals have been shaped through selective optimization to adapt to environmental constraints [@Bullmore2012; @Perin2011]. For example, in nematodes, the spatial arrangement of ganglia is optimized to minimize the total length of neural wiring [@White1986]. Similarly, in ctenophores, understanding how transmission via syncytial neurons and synaptic connections has been selectively optimized at the circuit level will provide crucial insights into the evolutionary scaling of neural networks from early nervous systems to complex brains [@Farnworth2024]. In particular, this research may help elucidate how circuit modularization and layered structures emerged over evolutionary time.

## Materials and Methods

### Specimen Preparation, Volume Electron Microscopy, and Image Processing

Larvae of *Mnemiopsis leidyi* (five days old) were cryofixed using a high-pressure freezing apparatus (HPM Live μ, CryoCapCell, https://www.cryocapcell.com/technologies/high-pressure-freezer) and immediately transferred to liquid nitrogen for storage. The frozen samples were processed in a substitution medium containing 2% (w/v) osmium tetroxide and 0.5% uranyl acetate in acetone, using a cryo-substitution device (EM AFS-2, Leica). Cryo-substitution was carried out by gradually raising the temperature from -90°C to -70°C over 4 hours, then returning to -20°C over 2 hours, and finally to room temperature over 2 hours. The samples were then embedded in epoxy resin (EMbed 812, Electron Microscopy Sciences). Serial ultrathin sections of 50 nm thickness were prepared using a Leica UC7 ultramicrotome and a 45º DiATOME diamond knife. To enhance section adhesion and improve hydrophilicity, a conductive indium tin oxide-coated glass slide (ITO Glass, UQG Optics) was treated with air glow discharge using the PELCO easiGlow system (Ted Pella, Inc.), rendering the surface negatively charged. Section ribbons were collected on prepared glass slides, gently dried to ensure proper stretching, and firmly adhered to the glass surface. The sections were stained with UranyLess and lead citrate (Reynolds) using airless staining procedures. The glass slides were mounted on a STEM-specific stage (Zeiss) using Copper Foil EMI Shielding Tape (3M). Imaging was performed using a Gemini SEM 500 (Zeiss), equipped with SmartSEM and Atlas 5 imaging software (Zeiss), using the Inlens detector. A total of 620 serial sections, each 50 nm thick, were analyzed. The imaging resolution was 2.8 nm/pixel, and the acceleration voltage was set to 1.5 kV. The imaging time for each tile was 54 minutes and 21 seconds, with a pixel size of 2.8 nm, a tile size of 91.8 × 91.8 µm (32768 × 32768 pixels), and a dwell time of 3.0 µs.

### Image-Stack Alignment and Export for CATMAID

To process the image stack, we utilized the TrakEM2 plugin of FIJI (ImageJ) (version 2.0.0-rc-15/1.49k / Java 1.6.0_24 (64-bit) – 2014). A project was created, and all TIFF images were imported using the "import sequence as grid" function. Subsequently, the following filters were applied sequentially: Invert, Equalize Histogram, and Gaussian Blur.

The alignment process consisted of three stages, each progressively refining the spatial accuracy (rigid, affine, elastic).

Initially, a rigid alignment was run with the following parameters: least squares mode (linear feature correspondences), encompassing the entire layer range with the first layer as the reference. Only visible images were used, without propagation. The alignment was executed with an initial Gaussian blur of 1.6 pixels, three steps per scale octave, a minimum image size of 512 pixels, and a maximum of 2048 pixels. Additional parameters included a feature descriptor size of 8, orientation bins of 8, and a closest ratio of 0.92. The alignment allowed clearing the cache, using 32 feature-extraction threads, a maximal alignment error of 100 pixels, a minimal inlier ratio of 0.20, and a minimum of 12 inliers. The expected and desired transformations were set to rigid, with testing multiple hypotheses (tolerance: 5.00 pixels) and considering up to 5 neighboring layers, giving up after 5 failures. Regularization was done with a maximal iteration of 1000, a maximal plateau width of 200, and a rigid lambda of 0.10.

Next we applied an affine alignment step with similar parameters, except the expected and desired transformations were set to affine. The minimal image size was reduced to 64 pixels, while the other parameters (Gaussian blur, feature descriptor size, inliers, and testing hypotheses) remained unchanged to ensure consistent processing.

Finally, we ran two iterations of elastic alignment to fine-tune the spatial data. Key parameters included a block-matching layer scale of 0.05, a search radius of 200 pixels, a block radius of 2000 pixels, and a resolution of 60. Correlation filters were employed with a minimal PMCC r of 0.10, a maximal curvature ratio of 1000, and a maximal second-best r/best r of 0.90. A local smoothness filter was applied with the approximate local transformation set to affine, a local region sigma of 1000 pixels, and an absolute maximal local displacement of 10 pixels (relative maximal displacement: 3.00). Pre-aligned layers were tested for up to 4 neighboring layers. The elastic alignment used a rigid approximation, maximal iterations of 3000, a plateau width of 200, spring mesh stiffness of 0.01, and a maximal stretch of 2000 pixels. A legacy optimizer was employed to enhance performance.

After each alignment stage, the project was saved as an XML file under a unique name to preserve iterative progress. Finally, the images were exported from FIJI using TrakEM2 in a format compatible with CATMAID.

### Neuron Tracing, Synapse Annotation, and Review

For skeletonisation, annotation and tagging we used CATMAID installed on a local server [@Saalfeld2009; @Schneider-Mizell2016]. To mark the locations of cell bodies, we placed tags at the approximate center of each nucleus within the dataset. At each nuclear center, the radius of the single node was adjusted according to the size of the cell body in that specific layer. All skeletons were rooted at their respective cell bodies, and the root nodes were tagged as "soma." Synapses were identified based on four key structural features: the cell membrane, synaptic vesicles, the endoplasmic reticulum, and mitochondria. Most synapses could be verified across consecutive sections, ensuring accurate annotation and connectivity mapping.

### Cell-type Nomenclature, Annotations and Connectome Analysis

We assigned each cell a specific cell-type name based on its category (balancer, bridge, bristle, ciliated grooves, dense vesicle cell, dome, epithelial floor (ef), intracellular multiciliated cells (imc), lamellate bodies (lb), lithocyte, plumose cell, ANN) resulting in a total of 12 cell types. Some cells only had more general features and were classified based on the presence/absence and number of cilia into four ciliated cell types:(biciliated (biC), monociliated (monoC), multiciliated (multiC), non-ciliated (non-C)). Additionally, we appended the quadrant number to the cell type name after an underscore ("\_") to indicate the cell-body’s location. For example, cells located in the first quadrant are named with "_Q1." If a cell is situated between the first and second quadrants, we appended "Q1Q2" to the cell-type name.

We further distinguished cells of the same type within the same region by serial numbering. This way each cell in the volume has a unique name string. Each cell was also assigned multiple annotations, which can be utilized to query the database via CATMAID or the CATMAID API (e.g., using the R catmaid package [@Bates2020]). These annotations provide a structured and precise framework for identifying and analyzing specific cells, facilitating robust data integration and retrieval from the dataset.

### Imaging the Activity of Balancer Cilia

For ciliary imaging, we used the cydippid stage of *M. leidyi* at five days post-fertilization. A coverslip with a thin layer of Vaseline applied to its two edges was gently placed over a glass slide. Filtered natural seawater and the cydippids were introduced beneath the coverslip. The orientation of the larvae was adjusted by carefully sliding the coverslip, and slight pressure was then applied to immobilize the larvae.

Balancer cilia movement was imaged using a differential interference contrast (DIC) microscope (Zeiss Axio Imager.M2) equipped with a high-sensitivity monochrome CMOS camera optimized for the near-infrared range (UI-3360CP-NIR-GL Rev.2, iDS), and a 40x glycerine-immersion objective lens (LD LCI Plan-Apochromat 40x/1.2 Imm Corr DIC M27). To stabilize the statolith, the microscope was tilted by 90° to achieve vertical stage alignment. Illumination was provided by a custom-made NIR LED strobe system (850 nm) synchronized with the camera’s exposure signals. Image acquisition was controlled using BohNavi software. Recordings were acquired at 640 × 480 pixels, 100 frames per second (fps), for a duration of 2 minutes, using 0.05 ms pulse illumination from the LED.

For beat frequency analysis, kymographs were generated using the Multi Kymograph plugin in Fiji. One-second segments were manually extracted from the kymographs, and ciliary beat frequencies were counted visually for each ROI. These manually determined frequencies were then used for subsequent computational analysis.

To process the beat frequency time series data, left and right cilia traces were extracted, filtered to remove missing or non-finite values, and assigned frame indices. Data were saved as RDS objects for downstream analysis. Mean values were plotted as bar graphs with the left and right cilia shown in positive and negative directions, respectively.

To assess coordination between left and right cilia, we computed rolling Pearson correlation coefficients over a window of 20 frames. For each dataset, we excluded windows with insufficient valid data or no variation. Time series of correlation coefficients were plotted and summarized as mean values. These mean correlations were then compared across planes (sagittal vs. tentacular) using Welch's t-test. The resulting statistics were visualized as boxplots with jittered data points.

### Data and Code Availability

The EM image stacks, including all traces and annotations, are available at https://catmaid.jekelylab.ex.ac.uk. The dataset encompasses all EM images (in JPG format), skeletons, meshes, node tags, connectors, and annotations. Additionally, we provide all R scripts used for data acquisition and figure generation (Jokura et al., 2025). All plots, figures (including anatomical renderings), and figure layouts can be fully reproduced using the provided R scripts. While the scripts are mostly organized by figure, some general-purpose scripts—for tasks such as loading libraries, accessing CATMAID data, and defining common functions—are shared across multiple figures.

## Acknowledgements

This work was supported by the Japan Society for the Promotion of Science (JSPS) Overseas Research Fellowships, the Grass Foundation, the Kavli Foundation, and the Marine Biological Laboratory (MBL). This project also received funding from the European Research Council (ERC) under the European Union’s Horizon 2020 research and innovation programme (grant agreement No. 101020792). We thank Dr. Réza Shahidi for sectioning and imaging, and Paulina Cherek for high-pressure freezing and sample preparation. We are also grateful to Dr. Chris Bjornsson, the MBL Central Microscopy Facility, Dr. Shoji A. Baba, and Dr. Kogiku Shiba (Shimoda Marine Research Center) for their assistance with microscopy.

## References {.unnumbered}

::: {#refs}
:::

```{=html}
<!--
## See https://github.com/hypothesis/client
## Add commenting functionality with this script
-->
```

```{=html}
<script src="hypothes.is/embed.js" async></script>
```

\newpage

## Supplementary material {.unnumbered}

![**Supplemental figure**\
(A)...](figures/Supplemental_figure.png)
