---
title: "Neural Circuit Organization of the Ctenophore Aboral Organ"

subtitle: |
  **Kei Jokura**$^{1,2,3,4*}$, **Sanja Jasek**$^{1,2,6}$, **Lara Kewalow**$^{6}$, **Pawel Burkhardt**$^{5}$, **Gáspár Jékely**$^{1,2,6,*}$  
  <br><br>  
  $^{1}$ Living Systems Institute, University of Exeter, Exeter, EX4 4QD, United Kingdom  
  <br>
  $^{2}$ Biosciences, Faculty of Health and Life Sciences, University of Exeter, Exeter EX4 4QD, UK
  <br>
  $^{3}$ Exploratory Research Center on Life and Living Systems (ExCELLS), Okazaki, 444-8787, Japan  
  <br>  
  $^{4}$ National Institute for Basic Biology (NIBB), Okazaki, 444-8585, Japan  
  <br>  
  $^{5}$ Michael Sars Centre, University of Bergen, Norway  
  <br>
  $^{6}$ Heidelberg University, Centre for Organismal Studies (COS), 69120 Heidelberg, Germany  <br><br>  
  **$^{*}$ Correspondence:**  
  [jokura@nibb.ac.jp](mailto:jokura@nibb.ac.jp),  
  [gaspar.jekely@cos.uni-heidelberg.de](mailto:gaspar.jekely@cos.uni-heidelberg.de)
mainfont: Arial
monofont: DejaVu Sans Mono #Times New Roman
fontsize: 11pt
linestretch: 1.2
linkcolor: blue
documentclass: article
max-width: 100%
format:
  html:
    embed-resources: true 
  pdf:
    pdf-engine: xelatex
    number-sections: false
    margin-left: 0.8in
    margin-right: 0.8in
    margin-top: 0.8in
    margin-bottom: 0.8in
    toc: false
    keep-tex: true
    include-in-header:
      text: |
        \usepackage{lineno}
    include-before-body:
      text: |
        \linenumbers
bibliography: references.bib #add bibliography in this file
csl: elife.csl #change according to journal
---

## Abstract

<!--# I rewrote a bit the abstract, please check -->

Ctenophores possess a unique gravity receptor (statocyst) in their aboral organ formed by four clusters of ciliated balancer cells that collectively support a statolyth. During reorientation, differential load on the balancer cilia leads to altered beating of the ciliated comb rows to elicit turns. To study the neural bases of gravity sensing, we imaged by volume electron microscopy (vEM) the aboral organ of the ctenophore *Mnemiopsis leidyi*. We reconstructed 909 cells, including syncytial neurons that form a nerve net and synapse on the balancer cells. Our connectome analysis revealed that these syncytial neurons establish direct and indirect pathways to balancer cells, with "bridge cells" acting as relay neurons. High-speed imaging revealed that balancer cilia beat and arrest in a coordinated manner but with differences between the sagittal and tentacular planes of the animal, reflecting nerve-net organisation. Given that ctenophores branched off early from other animals, these results inform our understanding of the early divergence of nervous systems.

## Introduction

<!--# the paper is not so much about evolution, so I suggest to start by introducing the cteno balancer organ -->

<!--# the things that need to be mentioned are -->

Ctenophores are gelatinous ... ciliary comb rows ... active swimming and reorientation ...

<!--# then introduce the nervous system of ctenos -->

Ctenophores show sophisticated and dynamic behavioral patterns and respond to a wide range of external stimuli. These behaviors are thought to be regulated by multiple neural systems, including the subepithelial nerve net (SNN), mesogleal neurons, an aboral organ, and tentacular nerves. However, the mechanisms by which these neural systems control behavior remain largely unexplored. Molecular phylogeny and chromosome-level synteny analyses suggest that ctenophores may be the sister group to all other animals [@Ryan2013; @Whelan2015a; @Whelan2017; @Whelan2023; @Li2021; @Schultz2023]. In agreement with this early divergence from all other animals with a nervous system, the ctenophore nervous system shows several unique features that suggest independent evolution from other animals <!--# cite some Moroz papers -->.

... syncytial nerve net ... highly peptidergic

The molecular makeup of ctenophore neurons is so divergent, that the first single-cell analyses did not identify a neuronal cluster [@Sebe-Pedros2018]. Only through the analysis of proneuropeptide markers were neuronal clusters identified [@Sachkova2021; @Hayakawa2022]. ... organisation largely unknown ... synapse structure

<!--# the phylogeny is not really relevant here, ctenos are interesting in their own right, so I would cut this a bit and we don't need to cite all the same genomic and tree papers, rater cite all the nice Tamm papers and EM papers etc -->

Another unique feature of ctenophores is their aboral organ that contains the statocyst. Ultrastructural analyses have revealed that the aboral organ contains a diverse array of sensory cells and neuronal components [@Hernandez-Nicaise1968; @Hernandez_Nicaise_1973; @Hernandez-Nicaise1974; @Tamm1982]. Immunostaining with an antibody labelling tyrosylated α-tubulin identified a neural structure called the deep nerve net within the aboral organ, suggesting its involvement in neural regulation of sensory-motor control [@Jager2011].

<!--# then the makeup and function of the balancer -->

...balancer in the aboral organ

The aboral organ, also known as aboral organ, of ctenophores contains a dense mineralized mass known as the statocyst, supported by four mechanosensory ciliated cells called balancer cells [@Noda_2014]. The ciliary movements of these balancer cells function as pacemakers for the eight rows of comb plates that control posture and swimming through mechanotransduction [@Tamm_2014; @Tamm1982]. Synaptic connections projecting to the balancer cells have also been reported [@aronova1974electron].

... balancer cilia, statolyth ... tuning of frequency ... pacemaker function ... reorientation

In this study, we combine volume EM and high-speed imaging to examine the gravity-sensing system within the aboral organ of the ctenophore *Mnemiopsis leidyi*. Through connectome analysis, we uncovered the structure and synaptic organisation of the aboral nerve net. Our imaging experiments revealed correlated changes in the activity of balancer cilia, suggesting a coordinating activity for the aboral nerve net.

## Results

### Volume EM reconstruction of the *Mnemiopsis* aboral organ

For vEM analysis, we used a five-day-old cydippid larva of *Mnemiopsis leidyi*. To obtain high-quality ultrastructural preservation, we used high pressure freezing followed by freeze substitution and resin embedding (Epon). The specimen was sectioned from the aboral tip and we collected approximately 1,000 ultra-thin (50 nm) serial sections as ribbons. The sections were imaged in a Zeiss Gemini 500 scanning electron microscope at 2.0 nm/pixel (xy). We imaged only the region containing the aboral organ. Subsequently, 620 of these images were stitched and aligned in TrakEM2 [@cardona2012]. The final vEM dataset encompassed a volume of 60 μm × 40 μm × 30 μm. We skeletonised and annotated all cells in this dataset in CATMAID [@Saalfeld2009], resulting in a reconstruction of 909 cells with nuclei and cell bodies intact. Most of the cells, including the balancers and the aboral nerve net neurons are fully contained within the volume, due to the compact and mostly self-contained organisation of the aboral organ.

In cells containing cilia, we also traced cilia along their length and annotated all basal bodies. For neuronal skeletonization, nodes were placed to interconnect the profiles of the same neuron’s processes across layers, extending the skeleton until all branches were fully traced. Each node was tagged <!--# is this true? not only a subset?  -->, and skeletons were named and assigned multi-level annotations. As described later, some neurons formed loop-like structures (anastomosed neurons), wherein separated branches often rejoined either the main branch or smaller branches [@Burkhardt2023]. In such cases, branch nodes were placed near the closest existing node and annotated accordingly (CATMAID only supports skeleton trees). The entire skeletonized volume was composed of 134,591 nodes. 88 fragments could not be attached to somata-associated skeletons. Most of these fragments represent short skeletal branches that could not be traced beyond gaps or low-quality layers.

Next, we divided the entire aboral organ into four broad quadrants (Q1-Q4) to facilitate grouping the identified cells (Figure 1F) [@Martindale1999]. The general body plan of ctenophores, when viewed from the aboral side, exhibits biradial symmetry around the anal pores [@Martindale1999]. This symmetry corresponds to the four blastomeres present at the four-cell stage during early embryonic development [@Martindale1999]. We assigned each of the 909 skeletonised cells to one of the four quadrants.

![**Figure 1. Morphology of the aboral organ in *Mnemiopsis leidyi***\
(A) Whole-body image of a 5-day-old *M. leidyi* cydippid larva in lateral view. The boxed region indicates the aboral organ (ao). Scale bar: 100 µm.\
(B) Schematic diagram of the aboral view of a cydippid larva. The two-tone coloration represents the biradial symmetry of the body. For convenience, the body is divided into four quadrants, designated as the first (Q1) through fourth (Q4) quadrants. The two primary viewing angles are referred to as the sagittal plane (S) and the tentacular plane (T). Abbreviations: ao, aboral organ; cg, ciliated groove; cr, comb plates; tb, tentacle bulbs.\
(C) Enlarged views of the aboral organ from different perspectives in a cydippid larva using a DIC microscope. The left panel presents an aboral view of the aboral organ. The middle panel shows a lateral view of the aboral organ in the sagittal (S) plane, while the right panel displays a lateral view in the tentacular (T) plane. Abbreviations: bal, balancer; cg, ciliated grooves; do, dome cilia; li, lithocyte. Scale bar: 50 µm.\
(D) Schematic representations of the aboral organ from three different perspectives. The left panel illustrates the aboral view, the middle panel shows the lateral view in the sagittal plane, and the right panel presents the lateral view in the tentacular plane. The quadrant colors correspond to those in panel (B).\
(E) Example of cell tracing using the collaborative annotation toolkit CATMAID for large-scale electron microscopy image datasets. The spherical objects indicate nuclear positions, while the lines represent the traced cell centers. Scale bar: 10 µm.\
(F) Three-dimensional reconstruction of cells composing the aboral organ, displayed from different perspectives. The left panel presents an aboral view of the aboral organ. The middle panel shows a sagittal plane view, while the right panel provides a tentacular plane view. Cells are color-coded according to their respective quadrants. Lithocytes (li) are represented as three gray spheres, while balancers (bal) are depicted as gray lines. Scale bar: 25 µm.](figures/Figure1.png)

### An aboral synaptic nerve net of syncytial neurons

Classical neural staining techniques did not provide clear images of neurons at the aboral organ. However, ultrastructural studies revealed morphological evidence for neural elements, including synapses that are located on the epithelial floor of the aboral organ—a region where epithelial cells are densely packed and form a floor-like structure [@Horridge_1964; @Hernandez_Nicaise_1973]. In our dataset, we identified synapses based on the previously described pre-synaptic triad morphology consisting of a single layer of vesicles, a cisterna of smooth endoplasmic reticulum and an associated mitochondrion. At synaptic sites, we marked mitochondria as synaptic nodes (orange in CATMAID) and connected this node to the nearest node in postsynaptic cells across the regions where synaptic vesicles aligned with the presynaptic cell's membrane (light blue arrows in CATMAID) <!--# figure.. -->. We could not detect specialized post-synaptic structures, in agreement with previous studies [@Hernandez_Nicaise_1973]. Synapses were identified as either monoadic or polyadic, with one pre-synaptic neuron forming connections with one or multiple post-synaptic cells <!--# figure of the distribution -->.

We reconstructed three aboral nerve-net (ANN) neurons, each with a syncytial morphology. Each ANN had multiple nuclei <!--# add numbers e.g. (2-5 nuclei) -->, with anastomosing membranes <!--# show example EM pictures -->. These neurons are distinct from the syncytial subepithelial nerve net (SNN) neurons with blebbed morphology previously identified by vEM in the body wall of *Mnemiopsis* [@Burkhardt2023]. Our results reveal a second type of syncytial neurons in ctenophores. <!--# describe the differences and also mention the classical SNNs that Sanja reconstructed -->

Furthermore, ANNs are also different from other neurons previously reported by EM, including mesogleal neurons and ciliated sensory cells that synalse on the SNN (types 1-4)[@Burkhardt2023]. Based on their distinct size and position, we classified the ANNs into two types. The first type is a single large neuron (ANN_Q1-4) spans all four quadrants (Q1-Q4) and has four (possibly five) nuclei This neuron has X <!--# add number --> presynaptic sites. The second type consists of two neurons (ANN_Q1Q2 and ANN_Q3Q4), each spanning two quadrants and with two nuclei, and each containing X presynaptic structures. <!--#  -->

![**Figure 2. Organisation of the aboral synaptic nerve net**\
(A) 3D reconstruction of the aboral nerve net (ANN) Q1Q2Q3Q4 (Q1-4), which spans all four quadrants and contains multiple nuclei (blue). The spheres represent the positions of individual nuclei. The left panel shows a aboral view of the aboral organ, the middle panel presents a sagittal plane view, and the right panel provides a tentacular plane view.\
(B) 3D reconstruction of two ANN Q1Q2 (pink) and Q3Q4 (orange), each spanning two quadrants and containing multiple nuclei. The spheres indicate the positions of the nuclei. The left panel shows a aboral view of the aboral organ, the middle panel presents a sagittal plane view, and the right panel provides a tentacular plane view.\
(C) Graph showing the number of mitochondria per cell and the proportion of mitochondria involved in synapse formation for each cell type. Green represents mitochondria associated with synapses, while gray represents mitochondria not involved in synapse formation. ANN has a significantly higher number of mitochondria compared to other cell types, with most of them forming synapses. The abbreviations are as follows: bal (balancer), brg (bridge), bsl (bristle), cg (ciliated groove), dv (dense vesicle cells), imc (intra-multiciliated cells), la (lamellate bodies), li (lithocytes), pl (plumose), and ef (epithelial floor cells).\
(D) Localization of mitochondria within ANN Q1-4. Red indicates mitochondria associated with the presynaptic triad structures, yellow marks mitochondria containing synaptic vesicles but lacking a clearly defined presynaptic triad, blue represents mitochondria with unclear synaptic vesicles, and black denotes mitochondria where no synaptic vesicles were identified.\
(E) Representative electron micrographs of presynaptic triad structures observed in the dataset and their schematic diagrams. The left diagram illustrates synaptic projections from the center ANN (ANN Q1-4) to the lateral ANN (ANN Q3Q4), while the right diagram shows synaptic projections from the lateral ANN (ANN Q1Q2) to the center ANN (ANN Q1-4). The abbreviations are as follows: mi (mitochondrion), er (endoplasmic reticulum), and sv (synaptic vesicles).\
(F) Plot showing the three-dimensional positions of synapses. Synapses from the center ANN to the lateral ANN are shown in magenta, while synapses from the lateral ANN to the center ANN are shown in cyan. Blue dots indicate the locations of autapses within ANN Q1-4.\
(G) Neural circuit map of ANN connectivity. The blue, orange, and pink circles represent individual neurons in ANN Q1-4, Q2Q3, and Q3Q4, respectively. Magenta arrows indicate synaptic connections from the center ANN to the lateral ANN, while cyan arrows indicate connections from the lateral ANN to the center ANN. Numbers denote the number of synapses. Blue arrows represent autapses in ANN Q1-4 along with their synapse counts.](figures/Figure2.png)

### Synaptic connectome of the gravisensory organ

Our synapse annotation revealed synapses between the ANN neurons and synapses that ANN_Q1-4 forms on itself (autapses). The ANN neurons also form synapses on the gravity-sensing balancer cells.

Balancer cells are monociliated cells with long motile cilia that form four bundles of compound cilia, one in each quadrant. These bundles come together at the centre of the aboral organ to support the cellular mass of the statolyth. The bending of the cilia and the position of somata differed clearly when viewed laterally from the sagittal or the tentacular plane (Figure 3B). In each quadrant, there were between 28-37 balancer cells (Q1: 37; Q2: 32; Q3: 32; Q4: 28). Each cell contained 3 to 10 <!--# check --> mitochondria.

ANN_Q1-4 formed synapses on balancer cells in all four quadrants (on 7 cells in Q1, 11 cells in Q2, 6 cells in Q3 and 10 cells in Q4) while ANN_Q1Q2 and ANN_Q3Q4 synapsed to balancer cells in their respective quadrants (ANN_Q1Q2 on 6/37 cells in Q1 and 8/32 cells in Q2; ANN_Q3Q4 on 1/32 cells in Q3 and 5/28 cells in Q4). Some balancer cells received inputs from both ANN_Q1-4 and either of ANN_Q1Q2 or ANN_Q3Q4. While previous studies suggested the presence of afferent synapses from balancer cells to neurons [@Hernandez-Nicaise1974], our data did not reveal any presynaptic sites in balancer cells.

The second group of cells that form synaptic contact with the ANN were the bridge cells. Bridge cells, first described by Tamm et al. in 2002 [@Tamm_2002], are characterized by bundles of elongated processes filled with microtubules that arch over the epithelial layer, resembling a bridge. Their somata are located at the base of the paired balancer-cell clusters along the tentacle surface and extend across the sagittal plane towards the base of the opposite balancer cells. Bridge cells form two distinct cell groups across the sagittal plane, in the Q1Q2 and Q3Q4 regions. In the Q1Q2 region, we identified 14 bridge cells, in Q3Q4, 12 cells.

Bridge cells had presynaptic sites with the typical presynaptic triad structure near 30% of their mitochondria. These bridge-cell synapses were formed on ANN neurons or other bridge cells. Bridge cells in the Q1Q2 region formed synapses on ANN_Q1Q2 (3 cells) or on both ANN_Q1Q2 and ANN_Q1-4 (2 cells). Bridge cells in Q3Q4 synapsed on ANN_Q3Q4 (1 cell) or ANN_Q1-4 (6 cells).

Nearly all bridge cells (25/26) also received synaptic input from ANNs. Bridge cells in Q1Q2 received inputs from ANN_Q1Q2 (11 cells), ANN_Q1-4 (1 cell), or both (2 cells). In Q3Q4, synapses on bridge cells were from ANN_Q3Q4 (1 cell), ANN_Q1-4 (7 cells), or both (1 cell).

In both The Q1Q2 and the Q3Q4 regions, bridge cells also formed synapses with adjacent bridge cells. However, we found no synapses across the sagittal plane to bridge cells in the opposite region.

To analyze the synaptic connectivity graph of the balancer organ and the ANN, we grouped cells of the same type and in the same region into a single node, summed the number of synapses, and laid out the network to reflect the anatomy of the four quadrants (Figure 3H). The network is characterised by feedback connections among ANN neurons and between ANN and bridge cells with a clear rotational (or mirror) symmetry. Q1Q2 and Q3Q4 also form separate local circuits centered around their regional ANN neurons while ANN_Q1-4 mediates global connectivity across the entire balancer organ. Notably, there were no synapses from the mechanosensory balancer cells to the ANN or from ANN to motor cells (e.g. ciliary groove), contrary to our initial expectation of a sensory-motor or input-output model of balancer function.

The ANN neurons also formed synapses on other cell types in the aboral organ, including the dense-vesicle cells, EF cells and several non-ciliated, monociliated or biciliated cells. These cell types and synaptic connections are outside the gravisensory organ and will be described elsewhere.

![**Figure 3. Reconstruction of the balancers and bridges that receives synapses from the ANN.**\
(A) Graph showing the number of synaptic outputs from the ANN circuit for each cell type. The balancer and bridge cell types receive relatively high outputs from the ANN circuit. ANN represents the number of synapses within the ANN circuit. The abbreviations are as follows: bal (balancer), brg (bridge), bsl (bristle), cg (ciliated groove), dv (dense vesicle cells), imc (intra-multiciliated cells), la (lamellate bodies), li (lithocytes), pl (plumose), and ef (epithelial floor cells).\
(B) Connectivity map of three ANN neurons (Q1-4 in blue, Q1Q2 in pink, and Q3Q4 in orange) and balancer ciliated cells (light blue). The thickness of the arrows and the numbers correspond to the number of synapses. Synaptic structures from the same neuron targeting the same balancer cell are grouped into hexagons, with the number of cells added to the label.\
(C) 3D reconstruction of bridge cells spanning the Q1Q2 and Q3Q4 quadrants. The Q1Q2-side bridge cells (8 cells) are shown in blue, while the Q3Q4-side bridge cells (6 cells) are shown in xxx color. The morphology of individual bridge cells extending across opposite quadrant regions is depicted. The spheres represent the positions of individual nuclei. The left panel shows an aboral view of the aboral organ, the middle panel presents a sagittal plane view, and the right panel provides a tentacular plane view.\
(D) Mitochondrial localization within bridge cells and associated presynaptic triad structures. Red indicates mitochondria associated with presynaptic triad structures, yellow marks mitochondria containing synaptic vesicles but lacking a clearly defined presynaptic triad, blue represents mitochondria with unclear synaptic vesicles, and black denotes mitochondria where no synaptic vesicles were identified. The left panel shows a dorsal view of the aboral organ, the middle panel presents a sagittal plane view, and the right panel provides a tentacular plane view.\
(E) Synaptic connections from ANN neurons to balancer ciliated cells. The positions of synapses from ANNs to balancers (magenta) are indicated. The left panel presents an aboral view, while the right panel shows a lateral view and a tentacular plane view. Balancer ciliated cells are depicted in light gray.\
(F) Synaptic connections between ANNs and bridge cells. The positions of synapses from ANNs to bridge cells (magenta) and from bridge cells to ANNs (light blue) are indicated. The left panel shows an aboral view, while the right panel presents a tentacular plane lateral view. Bridge cells are shown in light gray.\
(G) Connectivity matrix of the gravity-sensing neural circuit. Columns represent presynaptic cell groups, while rows represent postsynaptic cell groups. The numbers and varying shades of blue correspond to the number of synapses.\
(H) Complete connectivity map of the gravity-sensing neural circuit. Cells belonging to the same group are enclosed in hexagons, and the number of cells is added to their labels. The thickness of the arrows and the numerical values indicate the number of synapses. ANN Q1-4 are shown in blue, ANN Q1Q2 in orange, ANNs in Q3Q4 in pink, balancer cell groups in gray, and bridge cell groups in yellow.](figures/Figure3.png)

### Dynamics of balancer cilia imply a coordinating function for the nerve net

To investigate the regulation of balancer function, we carried out a high-speed video-microscope analysis of balancer cilia. Previous studies [@Tamm1980; @Tamm1982; @Lowe1997] have established that balancer cilia function as mechanoreceptors, with their beating frequency modulated by inclination. Tamm also suggested that differences in statolith morphology and the shape of balancer cilia between the tentacular and sagittal planes could lead to different forces exerted on cilia by the statolith [@Tamm_2014; @Tamm2015]. To further explore this, we used a tilted microscope with a vertical stage where we mounted immobilised cydippid larvae with their aboral-oral axis aligned in different orientations relative to the gravity vector. We then compared larvae oriented in different angles and either with their sagittal or tentacular plane parallel to the sample stage.

In larvae with their sagittal plane facing the objective, we could compared balancer-cilia movements between Q1(4) and Q2(3). In other larvae oriented in the tentacular plane, we could simultaneously image Q1(3) and Q2(4). We used a high-speed camera and recorded at 100 fps for 2 minutes (12,000 frames). To analyse ciliary beating, we selected regions of interest (ROIs) in areas where brightness changes indicated ciliary beating. Ciliary beating was both manually quantified and plotted as kymographs.

During the two-minute recordings, balancer cilia could beat fast, slow, or exhibit abrupt stops (arrest) and start moving again (re-beat). Occasionally, large body-contractions moved the entire cydippid out of frame, and data from these episods were excluded. We focused on three metrics for inter-quadrant comparison: (1) the timing of ciliary arrest, (2) the timing of re-beat after an arrested phase, and (3) ciliary beat frequency.

We found that arrest and re-beat events were synchronized between balancers across the sagittal plane. However, while the timing of re-beat was also near-simultaneous along the tentacular plane, arrest timing was offset by up to 2.xy <!--# add max value--> seconds (Figure 4) <!--# save source data for all plots in manuscript/source_data with the naming convention e.g. Figure 1A--source data 1 -->. Overall, these data reveal that arrests are only coordinated between Q1-Q2 and Q3-Q4, whereas re-beat is coordinated over all four quadrants.

Evaluating these data in light of the circuit diagram suggests that shared ANN inputs (ANN_Q1Q2 or ANN_Q3Q4) to a pair of balancers along the sagittal plane may underlie their synchronized arrests. In contrast, in the tentacular plane, separate ANNs innervate the balancer pairs. At the same time, the ANN_Q1-4 neuron synapses on all four balancers, hinting at a neural substrate for their synchronised re-beat.

![**Figure 4. Differences in Balancer Cilia Movement Control Due to Variations in the Neural Circuit Pathways and Comparison of Neural Circuits Regulating Ciliary Movement**\
(A) Schematic diagram of differential interference contrast (DIC) microscopy setup for imaging balancer cilia movement. The microscope was tilted 90 degrees so that the stage was positioned vertically. A monochrome CMOS camera sensitive to near-infrared (NIR) light was used, synchronized with an 850 nm strobe light source. The movement of the balancer cilia was recorded using a 40× objective lens.\
(B) Kymograph patterns comparing the arrest and re-beat of balancer cilia movement. The left and right balancer cilia movements are compared in both the sagittal plane (left) and tentacular plane (right). The direction and length of the arrows indicate time.\
(C) Graph showing the time differences in the arrest and re-beat timing of the left and right balancer cilia. S and T represent the sagittal plane and tentacular plane, respectively.\
(D) Each neuron represented by a circle is color-coded to indicate putative homologous cell types involved in ciliary movement regulation. Ciliated cells are shown in gray. Synapses are indicated by arrows, with magenta representing synapses that induce ciliary arrest and blue representing synapses that induce ciliary re-beat or an increase in ciliary beating frequency.\
(Left) Gravity-sensing neural circuit of *M. leidyi*, as revealed in this study. Bridge cells (yellow squares) are suggested to be electrically coupled (indicated by yellow zigzag lines), implying their potential involvement in feedback mechanisms between neurons and ciliated cells.\
(Righr) Neural circuit regulating prototroch ciliary movement in *P. dumerilii*. Serotonergic neurons (Ser-h1 neurons, blue) activate ciliary movement, while cholinergic neurons (MC neurons, magenta) induce ciliary arrest. Arrows directed toward ciliated cells in the head prototroch are color-coded accordingly.](figures/Figure4.png)

## Discussion

In this study, we traced all the constituent cells of the aboral organ in five-day-old larvae of *Mnemiopsis leidyi* and completed a comprehensive neural map. As a result, we identified a novel type of syncytial neuron with a previously unreported morphology. Furthermore, we found that three such syncytial neurons project to distinct quadrants, integrating with balancer cell groups to form a gravity-sensing neural circuit. This study represents the first complete neural circuit map reported in ctenophores. Additionally, our analysis of balancer ciliary movements, based on neural circuit data, revealed differences in ciliary arrest control arising from circuit variations.

### Identification of a Previously Unreported Syncytial Neuron

We identified three syncytial neurons with multiple nuclei in the aboral organ of five-day-old *M. leidyi* cydippid larvae. Burkhurdt et al. previously reported that in one-day-old larvae, the sub-epithelial nerve net (SNN) contains neurons that form syncytia [@Burkhardt2023]. However, the newly discovered syncytial neurons embedded within the aboral organ cell cluster did not exhibit the characteristic "pearl necklace-like structure" of the SNN, which consists of microtubules connecting neurons. Instead, their morphology was more reminiscent of fiber cells previously described in Placozoa [@Mayorova2021], with cell bodies extending through intercellular spaces. These morphological characteristics suggest that the newly identified neurons represent a second distinct type of syncytial neuron in ctenophores.

### Functional Diversification of the Three Syncytial Neurons

The three identified neurons may be classified into two distinct functional types based on their spatial arrangement and their influence on the differential control of balancer ciliary movement between the quadrants. In metazoans, a single neuron can directly provide synaptic input to multiple ciliated cells at a distance, synchronizing their movements. For instance, in larvae of the annelid *Platynereis dumerilii*, a cholinergic motor neuron (MC) innervates multiple prototroch ciliated cells, synchronizing their motion such that activation of the neuron leads to simultaneous arrest of ciliary beating [@Verasztó2017]. Additionally, a pair of serotonergic neurons, designated Ser-h1, extend axons that cross at the midline; the left Ser-h1_l predominantly innervates right-sided ciliated cells, while the right Ser-h1_r mainly projects to the left-side ciliated cells. Activation of Ser-h1 induces ciliary beating initiation and increases beat frequency.

In the this study, we found that three neurons are closely positioned within the confined space of the aboral organ, potentially enabling a single neuron to control distant ciliary movements in a synchronized manner. Moreover, the presence of neurons projecting to the same ciliated cells in distinct patterns suggests that these neurons may serve opposing functions—one inhibiting ciliary activity while the other promotes it, or one modulating ciliary beat frequency. While our synaptic ultrastructure analysis did not allow us to distinguish excitatory from inhibitory neurons, future electrophysiological investigations will be essential to elucidate their functional properties.

### Paracrine Signaling and Synaptic Transmission from Sensory to Neural Cells

Previous electron microscopy studies have suggested the presence of various sensory cells, including photoreceptor and mechanoreceptor cells, within the aboral organ of ctenophores [@HORRIDGE1965; @vinnikov1974e; @Krisch1973; Aronova1974]. By comparing our dataset with previously reported ultrastructural data, we successfully identified several sensory cells within the aboral organ. However, notably, we did not find any clear presynaptic structures between these sensory cells and either neural or other cellular targets. In prior studies, immunostaining for specific neuropeptides in ctenophores has revealed positive signals in distinct cell populations within the aboral organ [@Sachkova2021; @Hayakawa2022]. These findings suggest that many sensory cells in the aboral organ may primarily rely on neuropeptide-mediated paracrine signaling rather than direct synaptic connections, supporting the chemical brain hypothesis [@Jékely2021]. While paracrine transmission is generally slower than synaptic transmission, it allows for a greater diversity of information processing.

Conversely, previous research has demonstrated that mechanoreceptive sensory cells equipped with cilia (referred to as type 3 and type 4 cells) in ctenophores form direct synaptic connections with the sub-epithelial nerve net (SNN) and mesogleal neurons [@Burkhardt2023]. Based on their ultrastructural features, these sensory cells are believed to respond to physical stimuli such as water flow, vibrations, and direct contact, transmitting information rapidly to cilia and muscle cells [@Hertwig1880; @Chun1880; @Eimer1880].

Surprisingly, our study identified bridge cells as the primary cell type forming synaptic inputs onto neurons in the aboral organ. Bridge cells were first described by Tamm et al. based on their morphological characteristics, with cellular projections extending to the basal region of balancer cilia [@Tamm_2002]. Our findings suggest that bridge cells are electrically coupled to balancer cilia and provide rapid feedback to neurons via synaptic transmission. Future studies analyzing differential responses to various sensory stimuli will not only elucidate the functional roles of bridge cells but also provide insights into the evolutionary relationship between transmission speed and synapse development.

### Circuit Variability as a Driver of Behavioral Diversity

Our study suggests that differences in neural circuits contribute to variations in the timing of ciliary arrest, which may play a role in modulating swimming direction in ctenophores [@Satterlie2015a]. This finding provides evidence that increased circuit complexity contributes to behavioral diversification. By integrating multiple neurons and altering circuit structure, information can be branched, allowing for diverse motor patterns within the framework of fast synaptic transmission.

The neural cells identified in the aboral organ exhibit a divergent feedforward connectivity pattern, where a single neuron forms presynaptic connections with multiple downstream targets. Such a circuit design minimizes unwanted signal interference, reduces processing delays, and enables relatively fast response execution [@Luo2021]. Moreover, the presence of multiple divergent feedforward circuits suggests that slight variations in the synchronization and phase delay of ciliary beating (within the range of a few hundred milliseconds) may have evolved as a mechanism for generating behavioral diversity.

From a broader neuroanatomical perspective, nervous system structures across various animals have been shaped through selective optimization to adapt to environmental constraints [@Bullmore2012; @Perin2011]. For example, in nematodes, the spatial arrangement of ganglia is optimized to minimize the total length of neural wiring [@White1986]. Similarly, in ctenophores, understanding how transmission via syncytial neurons and synaptic connections has been selectively optimized at the circuit level will provide crucial insights into the evolutionary scaling of neural networks from early nervous systems to complex brains [@Farnworth2024]. In particular, this research may help elucidate how circuit modularization and layered structures emerged over evolutionary time.

## Materials and Methods

### Specimen Preparation, Transmission Electron Microscopy, and Image Processing

Larvae of *Mnemiopsis leidyi* (5 days old) were cryofixed using a high-pressure freezing apparatus (BAL-TEC HPM 010, Balzers) and immediately transferred to liquid nitrogen for storage. The frozen samples were processed in a substitution medium containing 2% (w/v) osmium tetroxide and 0.5% uranyl acetate in acetone, using a cryo-substitution device (EM AFS-2, Leica). Cryo-substitution was performed by gradually raising the temperature, and the samples were embedded in epoxy resin. Serial sections of 50 nm thickness were prepared using a Reichert Jung Ultracut E ultramicrotome and a 45º DiATOME diamond knife. To enhance section adhesion and improve hydrophilicity, a conductive indium tin oxide-coated glass slide (ITO Glass, UQG Optics) was treated with air glow discharge using the PELCO easiGlow system (Ted Pella, Inc.), rendering the carbon film surface negatively charged. Section ribbons were collected on the prepared glass slides, slowly dried to ensure proper stretching, and firmly adhered to the glass surface. The sections on glass were stained with UranyLess and lead citrate (Reynolds) using airless staining bottles (Delta Microscopies). The glass slides were mounted on STEM-specific stages (Zeiss) using Copper Foil EMI Shielding Tape (3M). Imaging was performed using a Gemini SEM 500 (Zeiss) equipped with SmartSEM and Atlas 5 imaging software (Zeiss). The full dataset consisted of 620 serial sections, each 50 nm thick. The imaging resolution was 2.8 nm/pixel, using electron holography transmission at an acceleration voltage of 1.5 kV (in-lens detector, dwell time: 3 µs).

### Image-Stack Realignment and Spatial Data Transformation in CATMAID

To process the image stack, we utilized the TrakEM2 plugin of FIJI (ImageJ) (version 2.0.0-rc-15/1.49k / Java 1.6.0_24 (64-bit) – 2014). A project was created, and all TIFF images were imported using the "import sequence as grid" function. Subsequently, the following filters were applied sequentially: Invert, Equalize Histogram, and Gaussian Blur.

The alignment process consisted of three stages, each progressively refining the spatial accuracy:

Rigid Alignment Initially, a rigid alignment was performed with the following parameters: least squares mode (linear feature correspondences), encompassing the entire layer range with the first layer as the reference. Only visible images were used, without propagation. The alignment was executed with an initial Gaussian blur of 1.6 pixels, three steps per scale octave, a minimum image size of 512 pixels, and a maximum of 2048 pixels. Additional parameters included a feature descriptor size of 8, orientation bins of 8, and a closest ratio of 0.92. The alignment allowed clearing the cache, using 32 feature extraction threads, a maximal alignment error of 100 pixels, a minimal inlier ratio of 0.20, and a minimum of 12 inliers. The expected and desired transformations were set to rigid, with testing multiple hypotheses (tolerance: 5.00 pixels) and considering up to 5 neighboring layers, giving up after 5 failures. Regularization was performed with a maximal iteration of 1000, a maximal plateau width of 200, and a rigid lambda of 0.10.

Affine Alignment Following the rigid alignment, an affine alignment was applied using similar parameters, except the expected and desired transformations were set to affine. The minimal image size was reduced to 64 pixels, while the other parameters (e.g., Gaussian blur, feature descriptor size, inliers, and testing hypotheses) remained unchanged to ensure consistent processing.

Elastic Alignment Two iterations of elastic alignment were performed to fine-tune the spatial data. Key parameters included a block matching layer scale of 0.05, a search radius of 200 pixels, a block radius of 2000 pixels, and a resolution of 60. Correlation filters were employed with a minimal PMCC r of 0.10, a maximal curvature ratio of 1000, and a maximal second-best r/best r of 0.90. A local smoothness filter was applied with the approximate local transformation set to affine, a local region sigma of 1000 pixels, and an absolute maximal local displacement of 10 pixels (relative maximal displacement: 3.00). Pre-aligned layers were tested for up to 4 neighboring layers. The elastic alignment used a rigid approximation, maximal iterations of 3000, a plateau width of 200, spring mesh stiffness of 0.01, and a maximal stretch of 2000 pixels. A legacy optimizer was employed to enhance performance.

After each alignment stage, the project was saved as an XML file under a unique name to preserve iterative progress. Finally, the images were exported from FIJI using TrakEM2 in a format compatible with CATMAID.

### Neuron Tracing, Synapse Annotation, and Review

The reconstruction of cells and connectivity was performed using CATMAID [@Saalfeld2009], analyzing a total of 619 serial sections. To digitally reconstruct all neurons within the serial TEM dataset, we utilized the collaborative web-based application CATMAID, installed on a local server [@Saalfeld2009; @Schneider-Mizell2016]. To mark the locations of cell bodies, we placed tags at the approximate center of each nucleus within the dataset. At each nuclear center, the radius of the single node was adjusted according to the size of the cell body in that specific layer. All skeletons were rooted at their respective cell bodies, and nodes were tagged as "soma." Synapses were identified based on four key structural features: the cell membrane, synaptic vesicles, the endoplasmic reticulum, and mitochondria. Most synapses could be verified across consecutive sections, ensuring accurate annotation and connectivity mapping.

### Cell-type Nomenclature, Annotations and Connectome Analysis

Each cell is assigned a specific type name based on its category (balancer, bridge, bristle, ciliated grooves, dense vesicle cell, dome, epithelial floor (ef), intracellular multiciliated cells (imc), lamellate bodies (lb), lithocyte, plumose cell, ANN) resulting in a total of 12 cell types, or those that could not be classified: presence and number of cilia (biciliated (biC), monociliated (monoC), multiciliated (multiC), non-ciliated (non-C)) in 4 ciliated cell types. Additionally, a quadrant number is appended to the cell type name following an underscore ("\_") to indicate the cell body’s location. For example, cells located in the first quadrant are annotated with "Q1." If a cell is situated between the first and second quadrants, the annotation "Q1Q2" is applied.

Cells of the same type within the same region are further distinguished by serial numbering. Each cell is also assigned multiple annotations, which can be utilized to query the database via CATMAID or the CATMAID API (e.g., using the R catmaid package). These annotations provide a structured and precise framework for identifying and analyzing specific cells, facilitating robust data integration and retrieval within the dataset.

This system ensures consistent and accurate tracking of cellular data, enabling comprehensive analysis of cellular structures and functions across regions.

### Imaging of Balancer Cilia activity

We conducted the analysis using the cydippid stage of *M. leidyi* at five days post-fertilization. A coverslip with a thin layer of Vaseline applied to two edges was lightly placed on a slide glass. Filtered natural seawater was introduced under the coverslip, along with the cy3dippids. By gently moving the coverslip, the desired orientation for observation was adjusted, and once positioned, slight pressure was applied to immobilize the cydippids. To stabilize the statolith's position, the microscope was tilted 90 degrees, arranging the stage vertically. The movement of the balancer cilia was observed using a differential interference contrast (DIC) microscope (Zeiss Axio Imager.M2) equipped with a highly sensitive CMOS monochrome camera optimized for the near-infrared (NIR) range (UI-3360CP-NIR-GL Rev.2, iDS). Observations were made using a 40x objective lens (Objective LD LCI Plan-Apochromat 40x/1.2 Imm Corr DIC M27) with glycerine immersion. For recording, we employed a custom-made NIR LED strobe illumination system (wavelength: 850 nm) synchronized with the camera's exposure signals. The camera and LED strobe system were operated using the Video Capture Software BohNavi. Images were recorded at a resolution of 640×480 pixels with a frame rate of 100 fps for 2 minutes, using 0.05 ms pulses from the LED. To analyze the ciliary beating of the balancer cilia, we utilized the Multi Kymograph project in Fiji software.

### Data and Code Availability

The following EM image stacks, including all traces and annotations, are available at https://catmaid.jekelylab.ex.ac.uk. These datasets can be queried either within CATMAID or through the CATMAID application programming interface (API) using the catmaid or natverse packages [@Bates2020]. The dataset encompasses all EM images (in JPG format), skeletons, meshes, node tags, connectors, and annotations. Additionally, we provide all R scripts used for data acquisition and figure generation (Jokura et al., 2025). All plots, figures (including anatomical renderings), and figure layouts can be fully reproduced using the provided R scripts. While the scripts are mostly organized by figure, some general-purpose scripts—for tasks such as loading libraries, accessing CATMAID data, and defining common functions—are shared across multiple figures.

## Acknowledgements

This work was funded by ... JSPS... This project has received funding from the European Research Council (ERC) under the European Union's Horizon 2020 research and innovation programme (grant agreement No 101020792).

## References {.unnumbered}

::: {#refs}
:::

```{=html}
<!--
## See https://github.com/hypothesis/client
## Add commenting functionality with this script
-->
```

```{=html}
<script src="hypothes.is/embed.js" async></script>
```

\newpage

## Supplementary material {.unnumbered}

![**Supplemental figure**\
(A)...](figures/Supplemental_figure.png)
